taichi/backends/metal/kernel_util.cpp

```diff
@@ -18,7 +18,7 @@ std::string KernelAttributes::buffers_name(Buffers b) {
   const static std::unordered_map<Buffers, std::string> m = {
       REGISTER_NAME(Root),
       REGISTER_NAME(GlobalTmps),
-      REGISTER_NAME(Args),
+      REGISTER_NAME(Context),
       REGISTER_NAME(Runtime),
```

Comment:
Thanks, a design very similar to current OpenGL backend! May do `_args_i32_` -> `_cntx_i32` refactor iapr.

Reply:
Actually there is no `_args_i32_` on the Metal backend. IIRC, OpenGL had to do this due to the lack of pointer castings? But Metal does have pointer types..